app/modules/node-app-service/locals.tf

```diff
@@ -0,0 +1,7 @@
+locals {
+  severity = {

```

Comment:
Can we have this as a variable instead of a local? Then we can define it in the respective environment files.

It's good practice to keep your config and your code separate. Can get messy if you have a bunch of environment specific things scattered around the code.

Reply:
I've removed the `locals.tf` and added severity levels as static values for now as we'll likely revisit these later after defining what types of alert constitute a verbose, low, medium, high, or critical alert: https://github.com/Planning-Inspectorate/infrastructure-environments/pull/49/commits/e7e087e9875a1f7e8117b93796466072eeb4be3c